### PR TITLE
I´ve add the CRUD controller of the companies

### LINKED ISSUE #14: Create controller for Company Branch
# REQUIREMENTS
We're going to desing and make functional the Manage Company Page

# Acceptance Criteria
List all required acceptance criteria
- [ ] Make functional the Create method
- [ ] Make functional the Read method
- [ ] Make functional the Update method
- [ ] Make functional the Delete method

### PR DESCRIPTION
**Pull request by SEVERO96**
Add the CRUD methods in the controller to be able to make the views
 in the admin dashboard

### CODE REVIEW

File: checador/app/controllers/companies_controller.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        @companies.save
+        redirect_to companies_path
+      end
+    
+      def edit
+        set_company
+      end
+    
+      def update
+        set_company
```
Comment (juliocabrera820) [inline checador/app/controllers/companies_controller.rb:22]: You can use before_action hook to call this method

File: checador/app/controllers/companies_controller.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      end
+    
+      def create
+        @companies = Company.new(company_params)
+        @companies.save
+        redirect_to companies_path
+      end
+    
+      def edit
+        set_company
```
Comment (juliocabrera820) [inline checador/app/controllers/companies_controller.rb:18]: You can use before_action hook to call this method

File: checador/app/controllers/companies_controller.rb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      end
+    
+      def update
+        set_company
+        @companies.update(company_params)
+      end
+    
+      def destroy
+        set_company
```
Comment (juliocabrera820) [inline checador/app/controllers/companies_controller.rb:27]: You can use before_action hook to call this method

**Comment by ManuelAcostaGuerrero** [comment]
Getting better every day, Excellent

File: checador/app/controllers/companies_controller.rb [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        redirect_to companies_path
+      end
+    
+    
+      private
+      def company_params
+        params.require(:company).permit(:name, :address)
+      end
+    
+      def set_Company
```
Comment (juliocabrera820) [inline checador/app/controllers/companies_controller.rb]: You have a typo must be lowercase the 'c'

**Review by SampabloC — DISMISSED** [review]

**Review by juliocabrera820 — DISMISSED** [review]
Nice job

File: checador/app/controllers/companies_controller.rb (hunk 4 above)
Comment (SEVERO96) [reply checador/app/controllers/companies_controller.rb]: ok.

**Review by ManuelAcostaGuerrero — DISMISSED** [review]

File: checador/app/controllers/companies_controller.rb (hunk 1 above)
Comment (SEVERO96) [reply checador/app/controllers/companies_controller.rb:22]: ok

**Review by RobertoCarMed — APPROVED** [review]

**Comment by RobertoCarMed** [comment]
Nice

**Review by YilKanda — APPROVED** [review]

**Review by martinmt-mx — APPROVED** [review]
All good!

**Pull request merged**